src/main/scala/Mathematics/AbsMin.scala

```diff
@@ -0,0 +1,22 @@
+package Mathematics
+
+object AbsMin {
+
+	/**
+	    * Method returns Absolute minimum Element from the list
+	    *
+	    * @param listOfElements
+    	    * @return
+    	*/
+	
+	def absMin(Elements : List[Int]): Int = {
+	    var length : Int = Elements.length;
```

Comment:
The same as for absMax, this time you use `minBy` instead of `maxBy`.

Reply:
done